### PR TITLE
os/OWNERS: use aliases for approvers and reviewers

### PR DESCRIPTION
**Pull request by jmarrero**
On https://github.com/openshift/ci-tools/tree/master/cmd/autoowners states:
`If the upstream does not have an OWNERS file, the utility will ignore syncing it.`

We want to use the ALIASES already defined on:
https://github.com/openshift/release/blob/master/OWNERS_ALIASES#L90

The PR to remove OWNERS from os is: https://github.com/openshift/os/pull/842

### CODE REVIEW

File: ci-operator/config/openshift/os/OWNERS [hunk 1]
```diff
@@ -1,47 +1,8 @@
 # DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
```
Comment (bgilbert) [inline ci-operator/config/openshift/os/OWNERS:1]: Is this still true?

Comment (jmarrero) [reply ci-operator/config/openshift/os/OWNERS:1]: I just copied what we have in the other files. But it does not seem true to me. Time to delete all comments in all files?

Comment (jmarrero) [reply ci-operator/config/openshift/os/OWNERS:1]: Unless ci-tools is what reconciles the OWNERS with ALIASES but not sure.

Comment (jmarrero) [reply ci-operator/config/openshift/os/OWNERS]: @bgilbert comments removed, will open another PR later to address the other files we own.

Comment (jmarrero) [reply ci-operator/config/openshift/os/OWNERS]: follow up: https://github.com/openshift/release/pull/29340

**Comment by bgilbert** [comment]
https://github.com/openshift/os/pull/842 merged.

/hold cancel

**Comment by cgwalters** [comment]
For reference this broke things.  Copy pasting from internal chat:

> TL;DR re: approve/lgtm access on openshift/os:

    OWNERS file in openshift/os is needed for lgtm/approve controls on that repo.  can contain aliases + non-ocp-org members.
    OWNERS file in openshift/release/ci-operator/{config,jobs} for openshift/os controls lgtm/approve acess on that subdir in openshift/release. automatically syncs with upstream (openshift/os) OWNERS; will filter out non-ocp-org users and expand aliases

(See https://github.com/openshift/installer/blob/master/OWNERS and https://github.com/openshift/installer/blob/master/OWNERS_ALIASES as example upstream configs)
